manager/src/test/java/com/redhat/developer/manager/ConnectorsServiceTest.java

```diff
@@ -0,0 +1,87 @@
+package com.redhat.developer.manager;
+
+import java.util.List;
+
+import javax.inject.Inject;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import com.redhat.developer.manager.models.Connector;
+import com.redhat.developer.manager.models.ConnectorStatus;
+import com.redhat.developer.manager.requests.ConnectorRequest;
+
+import io.quarkus.test.TestTransaction;
+import io.quarkus.test.junit.QuarkusTest;
+
+@QuarkusTest
+public class ConnectorsServiceTest {
+
+    @Inject
+    ConnectorsService connectorsService;
+
+    @Test
+    @TestTransaction
+    public void testGetEmptyConnectorsToDeploy() {
+        List<Connector> connectors = connectorsService.getConnectorsToDeploy();
+        Assertions.assertEquals(0, connectors.size());
+    }
+
+    @Test
+    @TestTransaction
+    public void testGetEmptyConnectors() {
+        List<Connector> connectors = connectorsService.getConnectors("jrota");
```

Comment:
Can you inject the `CustomerIdResolver` into the test class and call that to get the `id` rather than hard-coding `jrota` directly

Reply:
The `CustomerIdResolver` is just hardcoding `jrota` for the time being, but the idea is that we will use it to extract the customerId from the JWT token and use it in the platform. 
I think the tests should not call `CustomerIdResolver`, otherwise we will have to change it everywhere when we do this change

Reply:
Moved to a constant btw